Update README (#26)
musicglue/liquid-elixir: README.md

end
```

> You can start the application as a children to your app:

Reviewer: ```suggestion
You can start the application as a child to your app:
```

```elixir
children = [
  worker(Liquid.Process, [
    :liquid,
    Application.get_env(:your_app, :liquid)
  ])
]

opts = [strategy: :one_for_one, name: Hokusai.Supervisor]
Supervisor.start_link(children, opts)
```

where config is:

```elixir
config :your_app, :liquid,
  cache_adapter: Cachex,
  filter_modules: [
    ...
  ],
  extra_tags: %{
    tag_1: {TagModule, Liquid.Tag},
    block_1: {BlockModule, Liquid.Block},
  },
  file_system: {Filesystem, "/"}
```

Or start it manually:

``` elixir
{:ok, pid} = Liquid.Process.start_link(:liquid, options)
```

where option is a keyword list, same as in config.

> Every option can be overrided by a keyword list passed as last argument to `Liquid.parse_template` / `Liquid.render_template`.

Reviewer: ```suggestion
Every option can be overridden by a keyword list passed as last argument to `Liquid.parse_template` / `Liquid.render_template`.
```

Compile a template from a string:

`template = Liquid.parse_template(:liquid, "{% assign hello='hello' %}{{ hello }}{{world}}")`

Render the template with a keyword list representing the local variables:

`{ :ok, rendered, _ } = Liquid.render_template(:liquid, template, %{"world" => "world"})`

For registers you might want to use in custom tags you can assign them like this:

`{ :ok, rendered, _ } = Liquid.render_template(:liquid, template, %{"world" => "world"}, registers: %{test: "hallo")`

The tests should give a pretty good idea of the features implemented so far.

> All of public API can be viewed inside `Liquid` module.

Reviewer: ```suggestion
All of the public API can be viewed inside the `Liquid` module.
```

end
```

> and than include them in your configuration, as seen above.

Reviewer: ```suggestion
and then include them in your configuration, as shown above.
```

Another option is to set up the tag using:
`Liquid.register_tags(:liquid, "minus_one", MinusOneTag, Liquid.Tag)`,
`Liquid.register_tags(:liquid, "my_block", ExampleBlock, Liquid.Block)` same for blocks;
and for filters you should use
`Liquid.add_filters(:liquid, MyFilters)`

#### Global Filters

> It's also possible to apply global filter to all rendered variables setting up in the config:

Reviewer: ```suggestion
It's also possible to apply a global filter to all rendered variables setting up in the config:
```

``` elixir
[global_filter: &MyFilter.counting_sheeps/1]
```

> or adding a `"global_filter"` value to options for `Liquid.render_template` function:

Reviewer: ```suggestion
or adding a `"global_filter"` value to options for the `Liquid.render_template` function:
```

> `Liquid.render_template(:liquid, :tpl, %{some: context}, global_filter: &MyFilter.counting_sheeps/1)` (you need to define filter function first)

Reviewer: ```suggestion
`Liquid.render_template(:liquid, :tpl, %{some: context}, global_filter: &MyFilter.counting_sheeps/1)` (you need to define the filter function first)
```
